compiler/src/dmd/dsymbol.h

```diff
@@ -177,7 +177,7 @@ class Dsymbol : public ASTNode
 public:
     Identifier *ident;
     Dsymbol *parent;
-    Symbol *csym;               // symbol for code generator
+    void *csym;                  // symbol for code generator
```

Comment:
Same as mtype.h, can just skip on updating the header.

Reply:
done